guides/source/association_basics.md

```diff
@@ -761,15 +761,53 @@ class Book < ApplicationRecord
 end
 ```
 
-Active Record will attempt to automatically identify that these two models share a bi-directional association based on the association name. In this way, Active Record will only load one copy of the `Author` object, making your application more efficient and preventing inconsistent data:
+Active Record will attempt to automatically identify that these two models share
+a bi-directional association based on the association name. This information
+allows Active Record to:
+
+* Prevent needless queries for already-loaded data
+
+```irb
+irb> author = Author.first
+irb> author.books.all? do |book|
+irb>   book.author.equal?(author) # No additional queries executed here
+irb> end
+=> true
+```
```

Comment:
If we want to maintain the bulleted list margin, we should indent these code snippets:

```suggestion
    ```irb
    irb> author = Author.first
    irb> author.books.all? do |book|
    irb>   book.author.equal?(author) # No additional queries executed here
    irb> end
    => true
    ```
```